src/pages/MyTokens/NFTPage.tsx

```diff
@@ -128,19 +128,14 @@ export const NFTPage = () => {
   }, [onSearch]);
 
   const filter = useCallback((token: NFTToken & Partial<Offer>) => {
-      let filterByStatus = true;
-      if (filterState?.onSell) {
-        filterByStatus = !!token.seller;
-      }
-      if (filterState?.fixedPrice) {
-        filterByStatus = !!token.seller && !token.auction;
-      }
-      if (filterState?.timedAuction) {
-        filterByStatus = !!token.seller && !!token.auction;
-      }
-      if (filterState?.notOnSale) {
-        filterByStatus = !token.seller;
-      }
+      const filterByStatus = (token: NFTToken & Partial<Offer>) => {
```

Comment:
some comment on what is going on would be nice. Moving it to a function with better name is even better.